.gitignore

```
@@ -134,3 +134,11 @@ dmypy.json
 
 #sphinx docs
 _build
+
+#MC
+.vscode/settings.json
```

Thank you for adding this section to the .gitignore!
I noticed the files are still showing up (this can happen when they are added to the commit before updating the .gitignore), can you run `git rm --cached <file>` for each file and see if that gets rid of them?

---

Let's ignore the whole .vscode folder, actually.
I see that we have this folder in our repo right now, but let's remove it so people have the freedom to use any PEP 8 compliant formatter.
```suggestion
.vscode/
```

---

> Thank you for adding this section to the .gitignore! I noticed the files are still showing up (this can happen when they are added to the commit before updating the .gitignore), can you run `git rm --cached <file>` for each file and see if that gets rid of them?

Done.

---

> Let's ignore the whole .vscode folder, actually. I see that we have this folder in our repo right now, but let's remove it so people have the freedom to use any PEP 8 compliant formatter.

Done.

---

The .vscode folder is gone now 🥳 
There are still a few files that got left over, though. Try running this!
```
git rm --cached .project .pydevproject examples/metadata/metadata.json metadata/metadata.json
```

---

Oops! Just did.